test/e2e/testing-manifests/statefulset/mysql-upgrade/statefulset.yaml

```diff
@@ -140,7 +140,7 @@ spec:
   - metadata:
       name: data
       annotations:
-        volume.alpha.kubernetes.io/storage-class: default
+        volume.beta.kubernetes.io/storage-class: default
```

Comment:
@liggitt the alpha annotations doesn't work when I try this on my env. This should be beta annotations.

There are also other examples which still use alpha annotations. I'd like to send a PR to modify them. But I wonder if we still want to keep the beta annotations in 1.8? @jsafrane WDYT? If we want to remove it in 1.8, I can help remove it.

Reply:
sorry, I misread and thought these were the init container annotations you were referring to... I would keep these as-is and open an issue for @jsafrane / @kubernetes/sig-storage-bugs

Reply:
np.

Actually, both `init containers` and `storage class` graduated to GA in 1.6.

Reply:
ref: https://github.com/kubernetes/kubernetes/pull/42128